src/sentry/issues/auto_source_code_config/task.py

```diff
@@ -66,15 +64,16 @@ def process_event(project_id: int, group_id: int, event_id: str) -> None:
     if not stacktrace_paths:
         return
 
-    installation, organization_integration = get_installation(org)
-    if not installation or not organization_integration:
+    try:
+        installation = get_installation(org)
+    except InstallationNotFoundError:
         logger.info("No installation or organization integration found.", extra=extra)
         return
 
     trees = get_trees_for_org(installation, org, extra)
     trees_helper = CodeMappingTreesHelper(trees)
     code_mappings = trees_helper.generate_code_mappings(stacktrace_paths)
-    set_project_codemappings(code_mappings, organization_integration, project)
+    set_project_codemappings(code_mappings, installation, project)
```

Comment:
We don't need the `organization_integration`. We can get all the info we need from the `installation`.